src/core/auth/services/auth.service.ts

```diff
@@ -4,6 +4,7 @@ import { CreateUserDto } from '@core/user/dto';
 import { TokenService } from '@libs/security';
 import { RedisService } from '@libs/redis/services/redis.service';
 import { UserService } from '@core/user/services';
+import { UserAlreadyExistsException } from '@common/exceptions/user-already-exists-exception';
```

Comment:
```suggestion
import { UserAlreadyExistsException } from '@common/exceptions';
```
Should add `index.t`s to folder `common/exceptions`
Should rename as `user-already-exists.exception.ts`